### DISCUSSION TITLE
Bug with file.folder

### DISCUSSION THREAD

**Discussion by FrankCHL** [SilentVoid13/Templater#1470] [Help]
Hello i have trouble with this command
![image](https://github.com/user-attachments/assets/703ac450-513e-43e9-b9a5-4aa19955da10)
when i use
![image](https://github.com/user-attachments/assets/84be0142-30e5-432a-9ca3-438b90262e28)
i have this result
![image](https://github.com/user-attachments/assets/a57d7b01-96a6-43ef-afd6-b479aaa73e46)

**Comment by Zachatoo** [comment]
Hi! That is not how you use `tp.file.folder`, please refer to the [documentation](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#tpfilefolderabsolute-boolean--false).

```js
<% tp.file.folder() %>
```
